src/stylesheets/base/_config.scss

```diff
@@ -18,12 +18,20 @@ $colors: (
   coral-pink:           #EA4446,
   urSuchAPill-dark-pink:#CF555E,
   urSuchAPill-pink:     #E75F69,
-  gradient-background:  linear-gradient(#184D66 0%, #D3E7E2 100%)
+  gradient-background:  linear-gradient(#184D66 0%, #D3E7E2 100%),
+  focus-outline:        rgba(78, 141, 241, 0.5)
 );
 
 $screens: (
   sm:                   'min-width: 31.25rem', // 500px
+  SM:                   'max-width: 31.25rem',
+
   md:                   'min-width: 45rem', // 720px
+  MD:                   'max-wdith: 45rem',
+
   lg:                   'min-width: 60rem', // 960px
-  xl:                   'min-width: 81.25rem' // 1300px
+  LG:                   'max-width: 60rem',
+
+  xl:                   'min-width: 81.25rem', // 1300px
+  XL:                   'max-width: 81.25rem'
```

Comment:
not a fan of this naming convention...maybe `max-xl` and `min-xl` would be clearer but that's also just an opinion

Reply:
i should've known this would be universally reviled, but the max-widths were an afterthought and it was too late to change all the `lg` to `min-lg`

Reply:
You could name them like below, where the first 3 are your existing min-width media queries, and the "max" versions are the max-width media queries:
```
sm:
md:
lg:
sm-max:
md-max:
lg-max:
```